R/messages.R

```diff
@@ -181,3 +181,31 @@ var_ord_msg <- function(reordered_vars, moved_vars, verbose) {
     cli_h2("All variables in dataset are ordered")
   }
 }
+
+#' Utility for data Lengths
+#'
+#' @param max_length Dataframe with data and metadata length
+#' @param verbose Provides additional messaging for user
+#'
+#' @return Output to Console
+
+max_length_msg <- function(max_length, verbose) {
+  if (nrow(max_length) > 0) {
```

Comment:
Add assertion on parameters

```suggestion
  assert_data_frame(max_length)
  assert_choice(verbose, choices = .internal_verbose_choices)
  
  if (nrow(max_length) > 0) {
```